test/account.spec.js

```diff
@@ -0,0 +1,115 @@
+var MOCKUP, assert, Account;
+
+assert = require('chai').assert;
+Account = require('../account');
+
+MOCKUP = {
+  'good': {
+    'name': 'Cuenta Corriente',
+    'kind': 'checking',
+    'currency': 'national',
+    'balance': 449086,
+    'number': '000067899946'
+  },
+  'badKey': {
+    'balance': 449086,
+    'currency': 'national',
+    'name': 'Cuenta Corriente',
+    'kind': 'checking',
+    'uid': '000067899946'
+  },
+  'badValue': {
+    'balance': 449086,
+    'currency': 'national',
+    'name': 'Cuenta Corriente',
+    'kind': 'asdf',
+    'number': '000067899946'
+  },
+  'missingValue': {
+    'balance': 449086,
+    'currency': 'national',
+    'name': 'Cuenta Corriente',
+    'number': '000067899946'
+  }
+};
+
+describe('Module Account', function () {
+  describe('Set object', function () {
+    it ('should throw if a key is not properly set.', function () {
+      var fn = function () {
+        var acc = new Account(MOCKUP.badKey);
+      };
+      assert.throws(fn, Error);
```

Comment:
This should test wether the correct error is sent, meaning you should specify the error string. something like

```javascript
assert.throws(fn, "<error string>");
```